src/components/TimezoneSelect.vue

```diff
@@ -14,6 +14,7 @@
 
   const props = defineProps<{
     modelValue: string | null,
+    showTimestamp?: boolean,
```

Comment:
seems like a pretty unrelated change. Was this a bug you stumbled across?

Reply:
Ah no, it looks like git didn't do a great job fast forwarding this branch on [this commit](https://github.com/PrefectHQ/orion-design/commit/1773abab0ab1431dfcaabc1be3e5f39bc10ed57c)

Reply:
I think it's probably ok though because @stackoverfloweth is still listed as the co-author so it'll just get fast forwarded when we merge